tests/integration/components/bs-form-element-test.js

```diff
@@ -28,7 +28,7 @@ module('Integration | Component | bs form element', function (hooks) {
   };
 
   test('form is submitted if valid and validation success shown', async function (assert) {
-    assert.expect(1);
+    assert.expect(2);
```

Comment:
I'm surprised that we need to update the expected number of assertions. The new value seems to be correct. Does that mean the tests were broken since ever? 😱

Reply:
The `assert.expect()` calls were only added in this PR! (I had to fix them in a later commit of this PR). This was due to new eslint rules for qunit.

Reply:
Oh. Seems that I had reviewed one commit and not the entire PR. Shouldn't have done it quickly in between other things. 😬